cmd/expired-authz-purger2/config.json

```
@@ -0,0 +1,12 @@
+{
+    "expiredAuthzPurger2": {
+        "syslog": {
+          "stdoutLevel": 6
+        },
+        "dbConnectFile": "test/secrets/purger_dburl",
+        "maxDBConns": 10,
+        "gracePeriod": "168h",
+        "batchSize": 1000,
+        "debugAddr": ":8014"
```

```suggestion
        "debugAddr": ":8015"
```

8014 is already used by the existing purger